### ISSUE TITLE
DIS user documentation

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#1482]
We need to provide user documentation for the new DIS features.

DIS is a simulation protocol - it's a way for different computer simulations to talk to each other: https://en.wikipedia.org/wiki/Distributed_Interactive_Simulation

Frequently the simulations don't have a display - they just do the hard sums.  So, we're giving Debrief the ability to listen on the network for a running computer simulation, and to display a live picture of what is being simulated.

We're adding this feature for a group of users that use the ODIN simulator, and we're working with the ODIN supplier so that ODIN sends out an agreed set of messages, and we display those messages.

But, we won't be distributing ODIN.  That will make it difficult for you to test it.  Fortunately, for our own testing, we've created a small computer simulator that can send messages out across the network.  This can be used to demonstrate the new capability.

I was keeping the test simulator out of sight. But, now I'm tempted to make it visible, so that both you and other Debrief users can have a play with it.

I've pictured these blocks of user guidance:
* Part B
  * introduction (introduce DIS, explain what Debrief is doing)
  * connecting to DIS  (network address & port for existing simulation)
  * Running a DIS Simulation (provide path to executable & input file. start/stiop)
  * DIS tips & tricks
* New "Using Debrief for DIS Analysis" Tutorial
  * configure Debrief to control external simulator
  * configure Debrief to listen for DIS messages
  * use Debrief to run external simulator
  * view new data being displayed
  * edit input file for external simulator
  * view new data being displayed

There would also be benefit in producing a blog article & tutorial video covering the tutorial content.

**Comment by IanMayo** [comment]
I'm collating a new release that can be used to develop the user guidance.

It's mostly feature complete, and it includes some documentation from me - in both the User Guidance and the Reference sections of the document.  

I'll send a link once its up & running.

I'll collate a current copy of the DocBook sources for you to base your work on, and email this over to you.

**Comment by IanMayo** [comment]
Russ, there is one new feature, that was implemented after I wrote the DocBook content.

It's now possible for users to view the output of the simulator, in the console view.   Here is the updated Listener view:
![updatedui](https://cloud.githubusercontent.com/assets/1108513/13771705/ad05657a-ea87-11e5-99f8-28420d43d3df.png)

And here is an example of the output shown:
![updatedui](https://cloud.githubusercontent.com/assets/1108513/13771714/b8814946-ea87-11e5-994f-34b37f594536.png)

I think the main focus of your task is to write the tutorial - which fortunately you're doing in MS Word.  But, if you do make DocBook corrections / tidying then you are welcome to record this new feature, and update the UI screenshot.

No problem if you want me to do it though,
